xml/System.Drawing.Imaging.Effects/BlurEffect.xml

```diff
@@ -57,7 +68,13 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>
+          <para>Boolean value that specifies whether the bitmap expands by an amount equal to the blur radius. If</para>
+          <para>
+            <see langword="true" />, the bitmap expands by an amount equal to the radius so that it can have soft edges. If</para>
+          <para>
+            <see langword="false" />, the bitmap remains the same size and the soft edges are clipped.</para>
+        </summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the bitmap expands by an amount equal to the radius so that it can have soft edges. <see langword="false" /> if the bitmap remains the same size and the soft edges are clipped.</value>
```